src/elastic.directive.ts

```diff
@@ -64,6 +65,10 @@ export class ElasticDirective implements OnInit, OnDestroy, AfterViewInit {
   private setupTextarea(textareaEl: HTMLTextAreaElement) {
     this.textareaEl = textareaEl;
 
+    const paddingTop = window.getComputedStyle(this.textareaEl).getPropertyValue('padding-top').replace('px', '');
+    const paddingBottom = window.getComputedStyle(this.textareaEl).getPropertyValue('padding-bottom').replace('px', '');
+    this.totalVerticalPadding = +paddingTop + +paddingBottom;
+
```

Comment:
I'd prefer to make a few changes here.

- Let's cache the result of `getComputedStyle` in a variable so it isn't calculated twice.
- Let's use `parseInt()` instead of the unary `+` operator, as this is consistent with elsewhere in the file.